### PR TITLE
Switch to HEMTT v1

### PR DESCRIPTION
**Pull request by veteran29**
**When merged this pull request will:**
- title

### CODE REVIEW

File: include/a3/ui_f/hpp/defineResinclDesign.inc [hunk 1]
```diff
@@ -328,19 +328,19 @@
...
+// #define IDC_RSCDISPLAYSELECTISLAND_NAMEBACKGROUND	1004
+// #define IDC_RSCDISPLAYSELECTISLAND_ISLANDPANORAMA	1200
+// #define IDC_RSCDISPLAYSELECTISLAND_INTERLACING	1201
+// #define IDC_RSCDISPLAYSELECTISLAND_CLOUD1		1202
+// #define IDC_RSCDISPLAYSELECTISLAND_CLOUD2		1203
+// #define IDC_RSCDISPLAYSELECTISLAND_CLOUD3		1204
+// #define IDC_RSCDISPLAYSELECTISLAND_CLOUD4		1205
+// #define IDC_RSCDISPLAYSELECTISLAND_ISLANDGROUP	2300
+// #define IDC_RSCDISPLAYSELECTISLAND_BUTTONCONTINUE	2600
+// #define IDC_RSCDISPLAYSELECTISLAND_BUTTONCANCEL	2700
```
Comment (3Mydlo3) [inline include/a3/ui_f/hpp/defineResinclDesign.inc:343]: Out of curiosity, why is this commented out? The same in `include/a3/ui_f/hpp/defineResincl.inc`. Was it removed?

**Review by 3Mydlo3 — APPROVED** [review]
Nice. Will we bump to 1.8.0 or any newer than 1.0.0?

File: include/a3/ui_f/hpp/defineResinclDesign.inc (hunk 1 above)
Comment (veteran29) [reply include/a3/ui_f/hpp/defineResinclDesign.inc:343]: HEMTT 1.9 warns about the redefinition of defines, in this file these defines are overwritten in some lines below:
https://github.com/ArmaForces/Mods/blob/de4b10e52939695a0a4a65385d81f5f8b31865f8/include/a3/ui_f/hpp/defineResinclDesign.inc#L714-L728

**Comment by veteran29** [comment]
> Nice. Will we bump to 1.8.0 or any newer than 1.0.0?

This adds supports for whole v1 in general. This branch builds fine with 1.9 (which added the duplicate macro checks)

**Pull request merged**